pages/style-guide/html.md

```diff
@@ -70,8 +116,61 @@ Don't indent inside `html`, `body`, `script`, or `style`. Indent inside `head` a
 </head>
 <body>
 
-<p>jQuery is awesome!<p>
+<h1 style="font-size:1em;line-height:2em;">This is a heading.</h1>
+<p style="text-decoration:underline;font-size:.5em;line-height:1em;">This is an underlined paragraph.</p>
+</body>
+</html>
+
+<!-- Good Example -->
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Sample Page</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+<body>
 
+<h1>This is a heading.</h1>
+<p>This is an underlined paragraph.</p>
 </body>
 </html>
 ```
+
+## Forms
+
+ - Always use `for` attribute in `label` element. Using `for` attribute in `label` element makes it accessible to screen readers.
```

Comment:
This isn't just about screen readers. This also doesn't address the concern about implicit labeling.